packages/framework/src/Console/Commands/BuildSiteCommand.php

```diff
@@ -77,7 +78,13 @@ protected function runPreBuildActions(): void
 
     public function runPostBuildActions(): void
     {
-        $service = new BuildTaskService($this->output);
+        $service = app(BuildTaskService::class);
+        $service->setOutput($this->output);
+
+        $service->registerIf(GenerateSitemap::class, $this->canGenerateSitemap());
+        $service->registerIf(GenerateRssFeed::class, $this->canGenerateFeed());
+        $service->registerIf(GenerateSearch::class, $this->canGenerateSearch());
+        $service->registerIf(GenerateBuildManifest::class, config('hyde.generate_build_manifest', true));
```

Comment:
We could also make so that these are registered in the config, as an alternate way of overloading (see https://github.com/hydephp/develop/issues/1150) but I think the use case for this is relatively slim, and the automatic overloading being implemented is nicer.